scripts/us_fed/treasury_constant_maturity_rates/extract_data.py

```diff
@@ -0,0 +1,41 @@
+USAGE = '''
+python3 extract_data.py [path]
```

Comment:
Files should start with at least a one line description comment.

Reply:
Fixed